frontend/javascripts/oxalis/api/api_latest.ts

```diff
@@ -636,23 +643,22 @@ class TracingApi {
       for (let y = min[1]; y < max[1]; y++) {
         for (let x = min[0]; x < max[0]; x++) {
           const id = data[idx];
-          if (!segmentIdToPosition.has(id)) {
+          if (!segmentIdToPosition.has(id) && id !== 0) {
```

Comment:
```suggestion
          if (id !== 0 && !segmentIdToPosition.has(id)) {
```

it's better to do the faster quick first for a better early out :)